numpy/linalg/linalg.py

```diff
@@ -2006,7 +2006,7 @@ def _multi_svd_norm(x, row_axis, col_axis, op):
     """
     if row_axis > col_axis:
         row_axis -= 1
-    y = rollaxis(rollaxis(x, col_axis, x.ndim), row_axis, -1)
+    y = moveaxis(x, [row_axis, col_axis], [-2, -1])
```

Comment:
Also, the tests are failing here. travis says

> ValueError: repeated axis in `source` argument

so I'm guessing this occurs when `row_axis` and `col_axis` are equal. Perhaps we _do_ need two `moveaxis` here.